styles/variables.less

```diff
@@ -162,8 +162,8 @@
 @sand-picker-font-size:             54px;
 @sand-picker-joined-font-size:      66px;
 @sand-steps-step-number-font-size:  54px;
-@sand-steps-step-dot-font-size:     30px;
-@sand-steps-step-current-dot-font-size:  48px;
+@sand-steps-step-highlightCurrent-font-size:         24px;
+@sand-steps-step-highlightCurrent-current-font-size: 42px;
```

Comment:
Shouldn't it be 30px?

Reply:
According to the guide, 30px refers to the size of the box, not the font size of the icon. So I modified the font size so that the circular icon can fit into the div.

Reply:
For the record, we do need to override the font size to meet the GUI guide since the current icon has much spaces than it's size.
The current icon looks like below with size 30.
![image](https://github.com/user-attachments/assets/6347533c-53e0-4e05-9db5-e4909edaf1c8)
This PR overrides the font size and mrgin so it looks like below.
![image](https://github.com/user-attachments/assets/05b7f5e2-c024-4700-9adb-5f3cf32b4c43)